### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by carrickr**
Prep scrubber for 5.x release

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -72,15 +73,15 @@
   end
 
   gem 'avalon-about', git: "https://github.com/avalonmediasystem/avalon-about.git", tag: 'avalon-r4'
-  gem 'about_page', git: "https://github.com/avalonmediasystem/about_page.git", tag: 'avalon-r4'
+  gem 'about_page', git: "https://github.com/avalonmediasystem/about_page.git"
```
Comment (cjcolvar) [inline Gemfile]: This eventually needs to have a tag because the latest about_page revs in master are for avalon-6 only.

File: Gemfile [hunk 2]
```diff
@@ -72,15 +73,15 @@
...
 
   # You are free to implement your own User/Authentication solution in its place.
   gem 'devise', '~>3.2.0'
   #gem 'devise-guests'
   gem 'haml'
 
   gem 'active_encode', git: "https://github.com/projecthydra-labs/active_encode.git", tag: 'v0.0.3'
-  gem 'rubyhorn', git: "https://github.com/avalonmediasystem/rubyhorn.git", tag: 'avalon-r5'
+  gem 'rubyhorn', git: "https://github.com/avalonmediasystem/rubyhorn.git"
```
Comment (cjcolvar) [inline Gemfile]: Same is probably true here that it should be tied to a tag.

File: app/models/api_token.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'securerandom'
+
+class ApiToken < ActiveRecord::Base
```
Comment (cjcolvar) [inline app/models/api_token.rb]: Do we really want to pull in this class and all of the changes around this?  I think not, but I could be wrong.

File: config/application.rb [hunk 4]
```diff
@@ -15,6 +15,7 @@ module Avalon
   class MissingUserId < Exception; end
 
   class Application < Rails::Application
+    require 'avalon/configuration'
```
Comment (cjcolvar) [inline config/application.rb]: I don't think we need the ENV configuration for this release either.

Comment (mbklein) [inline config/application.rb]: Letting ENV configuration merge and then testing it out with existing file configs is probably easier than excising it via cherry-pick. ENV configuration was written to be backward compatible with existing files.

**Pull request merged**